src/main/java/org/apache/sling/jcr/repoinit/impl/AclUtil.java

```diff
@@ -567,7 +567,7 @@ private static String toString(JackrabbitAccessControlEntry entry) throws Reposi
 
     private static void checkState(boolean expression, String msgPattern, Object... args) {
         if (!expression) {
-            if (args != null) {
+            if (args == null) {
```

Comment:
I don't get why this is changed. Can you please explain?

Reply:
Because it was wrong. I noticed the messages with the unsubstituted message patterns while investigating issues caused by the re-ordering. I didn't think this small fix was worth its own JIRA ticket.